src/moscot/backends/ott/solver.py

```diff
@@ -456,6 +456,13 @@ def _prepare(
             geom_kwargs["cost_matrix_rank"] = cost_matrix_rank
         geom_xx = self._create_geometry(x, t=time_scales_heat_kernel.x, is_linear_term=False, **geom_kwargs)
         geom_yy = self._create_geometry(y, t=time_scales_heat_kernel.y, is_linear_term=False, **geom_kwargs)
+        if alpha is None:
+            alpha = 1.0 if xy is None else 0.5  # set defaults according to the data provided
+        if alpha <= 0.0:
```

Comment:
also raise error if `alpha>1.0`

Reply:
Actually https://github.com/theislab/moscot/blob/f303a14ac47b337b099b43800e567c89ca7b3dd3/src/moscot/backends/ott/_utils.py#L91 gives the error for invalid range. I think it's better if one source only throws error. And thinking of it I think even the check in 461 is redundant so I can remove it

Reply:
ok makes sense!